docs/tutorials/release-using-channels.md

```diff
@@ -36,9 +37,9 @@ All you have to do to release using channels is to define the channel in the ver
 
 
 ### Your Application
-All you need to do here is to define which channel the user will receive.
+All you need to do here is to define which channel the user will receive with:
 
-`autoUpdater.channel('beta')`
+`autoUpdater.channel('beta')` (see [the documentation here](../auto-update.md#module_electron-updater.AppUpdater+channel))
```

Comment:
I have been doing `autoUpdater.channel = 'beta'`. Is this valid as well ?

Reply:
@kwent my bad.. channel('beta') is false. I will correct this.